src/handy-work.js

```diff
@@ -8,13 +8,18 @@ function loadPose(name, url) {
 	return HandPose.loadPose(name, url);
 }
 
+// Add event listeners
+let listenersLoaded = false;
+window.addEventListener('enter-vr', resetHands);
```

Comment:
I think these are AFrame specific events perhaps hooking into the Reset event onsession maybe the fix we need: https://immersive-web.github.io/webxr/#eventdef-xrreferencespace-reset